sys/include/net/gnrc/rpl/structs.h

```diff
@@ -238,7 +238,8 @@ struct gnrc_rpl_parent {
 typedef struct {
     uint16_t ocp;   /**< objective code point */
     uint16_t (*calc_rank)(gnrc_rpl_parent_t *parent, uint16_t base_rank); /**< calculate the rank */
-    gnrc_rpl_parent_t *(*which_parent)(gnrc_rpl_parent_t *, gnrc_rpl_parent_t *); /**< compare for parents */
+    gnrc_rpl_parent_t *(*which_parent)(gnrc_rpl_parent_t *, gnrc_rpl_parent_t *); /**< retrieve the better parent */
+    int (*parent_cmp)(gnrc_rpl_parent_t *, gnrc_rpl_parent_t *); /**< compare for parents */
```

Comment:
I know it hasn't been done before for the other function pointers in this file, but can you add – somehow – a documentation on how to interpret the return value of this function pointer? Is it possible to document function pointers in structs as if they were normal functinos with `@param, @return` etc.? Would be awesome.